detekt.yml

```diff
@@ -78,7 +80,7 @@ formatting:
     active: true
     autoCorrect: true
   OptionalUnit:
-    active: true
+    active: false
```

Comment:
Ideally we'd need the opposite option: flagging places where Unit isn't there. For us explicitness is key and we've been bitten by returns that should be Unit instead before.

Reply:
Agreed, If detekt rules can be user created we can also provide some to encourage a more functional style.